src/open_inwoner/conf/parts/maileditor.py

```diff
@@ -410,6 +410,67 @@
             },
         ],
     },
+    "case_status_notification_action_required": {
+        "name": _("Case status update notification (action required)"),
+        "description": _(
+            "This email is used to notify people about a new status being set on their case "
+            "that requires action on their part."
+        ),
+        "subject_default": "Action required",
+        "body_default": """
+            <p>Beste</p>
+
+            <p>U ontvangt deze email, omdat de status van een van uw zaken is bijgewerkt.</p>
+
+            <table>
+                <tr>
+                    <th>Zaakidentificatie</th>
+                    <td>{{ identification }}</td>
+                </tr>
+                <tr>
+                    <th>Zaaktype</th>
+                    <td>{{ type_description }}</td>
+                </tr>
+                <tr>
+                    <th>Startdatum</th>
+                    <td>{{ start_date }}</td>
+                </tr>
+            </table>
+
+            <p><a href="{{ case_link }}">Ga naar uw zaak</a> </p>
+
+            <p>Met vriendelijke groet,
+            {{ site_name }} </p>
+       """,
+        "subject": [
+            {
+                "name": "site_name",
+                "description": _("Name of the site."),
+            },
+        ],
+        "body": [
+            {
+                "name": "identification",
+                "description": _("The identification of the case"),
+            },
+            {
+                "name": "type_description",
+                "description": _("The description of the type of the case"),
+            },
+            {
+                "name": "start_date",
```

Comment:
Looking[ at these designs](https://www.figma.com/file/iKGhWhstaLIlFSaND2q7cE/OIP---Designs-(new)?type=design&node-id=1236-14623&mode=design&t=W9zC0gdJH18knf2X-0), we would also need some more back-end data for this:

- the required deadline/end_date of the case because of the text "Upload de documenten tot **13 februari 2024**"

Reply:
created a task for this https://taiga.maykinmedia.nl/project/open-inwoner/task/2077?